cypress/e2e/marc/marc-authority/edit-linked-authority-record/marc-authority-edit-linked-without-010-field-when-001-field-value-set-to-0.cy.js

```diff
@@ -0,0 +1,154 @@
+import Permissions from '../../../../support/dictionary/permissions';
+import TopMenu from '../../../../support/fragments/topMenu';
+import Users from '../../../../support/fragments/users/users';
+import DataImport from '../../../../support/fragments/data_import/dataImport';
+import Logs from '../../../../support/fragments/data_import/logs/logs';
+import JobProfiles from '../../../../support/fragments/data_import/job_profiles/jobProfiles';
+import getRandomPostfix from '../../../../support/utils/stringTools';
+import MarcAuthority from '../../../../support/fragments/marcAuthority/marcAuthority';
+import MarcAuthorities from '../../../../support/fragments/marcAuthority/marcAuthorities';
+import TopMenuNavigation from '../../../../support/fragments/topMenuNavigation';
+import InventoryInstances from '../../../../support/fragments/inventory/inventoryInstances';
+import InventoryInstance from '../../../../support/fragments/inventory/inventoryInstance';
+import QuickMarcEditor from '../../../../support/fragments/quickMarcEditor';
+import { JOB_STATUS_NAMES } from '../../../../support/constants';
+
+describe('MARC -> MARC Authority -> Edit linked Authority record', () => {
+  const testData = {
+    tag010: '010',
+    tag100: '100',
+    tag240: '240',
+    authority001FieldValue: '4284518',
+    authority100FieldValue: 'Beethoven, Ludwig van (no 010)',
+    searchOption: 'Keyword',
+    fieldForEditing: { tag: '380', newValue: '$a Variations TEST $2 lcsh' },
+    calloutMessage: 'Record cannot be saved with more than one 010 field',
+  };
+
+  const createdRecordIDs = [];
+
+  const marcFiles = [
+    {
+      marc: 'marcBibFileForC375139.mrc',
+      fileName: `testMarcFileC375139.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create instance and SRS MARC Bib',
+      instanceTitle: 'Variations / C375139Ludwig Van Beethoven.',
+      numOfRecords: 1,
+    },
+    {
+      marc: 'marcAuthFileForC375139.mrc',
+      fileName: `testMarcFileC375139.${getRandomPostfix()}.mrc`,
+      jobProfileToRun: 'Default - Create SRS MARC Authority',
+      authorityHeading: 'Beethoven, Ludwig van (no 010)',
+      numOfRecords: 1,
+    },
+  ];
+
+  before('Create test data', () => {
+    cy.loginAsAdmin({
```

Comment:
Please add deleting marc auth record before
cy.getAdminToken();
      // make sure there are no duplicate authority records in the system
      MarcAuthorities.getMarcAuthoritiesViaApi({ limit: 100, query: 'keyword="C375139"' }).then(
        (records) => {
          records.forEach((record) => {
            if (record.authRefType === 'Authorized') {
              MarcAuthority.deleteViaAPI(record.id);
            }
          });
        },
      );
And add to the marcAuthFileForC375139.mrc in the field 100 $a id 'C375139'. It helps deleting duplicates of auth records in DB